docs/manual/src/internals/ffi_converter_traits.md

```diff
@@ -0,0 +1,68 @@
+## Rust FFI conversion traits
+
+UniFFI leverages a set of FFI converter traits to implement [lifting and lowering](./lifting_and_lowering.md) on the Rust side.
+Each trait handles a single step in the lifting/lowering process (e.g. lifting an argument, lowering a return, etc.).
+We implement the trait for each type used in the exported API then use that trait in the codegen.
+
+For example, `uniffi::Lift` is used to lift values.
+To handle a function like `fn print(msg: String)`, the generated code will use:
+
+- `<String as Lift<crate::UniFfiTag>>::FfiType` when it needs to specify the FFI type (`RustBuffer`).
+- `<String as Lift<crate::UniFfiTag>>::try_lift()` when it needs to lift an argument value.
```

Comment:
maybe end with "(ie, in this example, to take an FFI value of type `<String as Lift<crate::UniFfiTag>>::FfiType` (as above, a`RustBuffer`) and convert it into a Rust string suitable for passing to the Rust function)" or similar?